src/test/ui/in-band-lifetimes/ellided_lifetimes.rs

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
```

Comment:
Add the .stderr file for the ui test

Reply:
You can run ./x.py test --stage 1 src/test/ui to get the stderr file

Reply:
Ya thanks. Just have to work on some stuff before that